FFXIVClientStructs/FFXIV/Client/UI/NumberArray/PartyListNumberArray.cs

```diff
@@ -0,0 +1,72 @@
+namespace FFXIVClientStructs.FFXIV.Client.UI.NumberArray;
+
+[NumberArray(4)]
+[StructLayout(LayoutKind.Explicit, Size = 0x04 * 727)]
+public unsafe partial struct PartyListNumberArray {
+    public const int PartyMemberNumberCount = 42;
+
+    [FieldOffset(0x04 * 002)] public int PartyLeaderIndex;
+    [FieldOffset(0x04 * 005)] public int PartyMemberCount;
+    
+    [FixedSizeArray<PartyMemberNumberArray>(8)]
+    [FieldOffset(0x04 * 009)] public fixed byte PartyMember[0x04 * PartyMemberNumberCount * 8];
+
+    [FieldOffset(0x04 * (009 + PartyMemberNumberCount * 0))] public PartyMemberNumberArray PartyMember0;
```

Comment:
Aren't these all just all redundant with the FixedSizeArray above?:
`PartyMemberSpan[0]`